packages/pages/src/common/src/template/types.ts

```diff
@@ -98,6 +98,24 @@ export interface TemplateConfig {
   alternateLanguageFields?: string[];
   /** The name of the onUrlChange function to use. */
   onUrlChange?: string;
+  /** Locales for a static page */
+  locales?: string[];
+}
+
+/**
+ * Shape of TemlateConfig for Static Pages
+ *
+ * @public
+ */
+export interface StaticTemplateConfig {
```

Comment:
@mkilpatrick Would we rather have this as a `type` that picks from `TemplateConfig` instead to reduce repetition? Or is something like this OK?

Reply:
This is ok in case they diverge a lot.